### DISCUSSION TITLE
jax source code that supports python3.6 version on github

### DISCUSSION THREAD

**Discussion by zmj101002** [jax-ml/jax#11555] [Q&A]
Dear jax team：
      Because of the requirements of the project, I need to build jax from source code on our platform, but  we only support python3.6, I would like to ask if you will provide jax source code that supports python3.6 on github,

**Comment by jakevdp** [comment]
Hi - thanks for the question. JAX dropped support for Python 3.6 in [version 0.2.18](https://jax.readthedocs.io/en/latest/changelog.html#jax-0-2-18-july-21-2021), in accordance with the established [deprecation policy](https://jax.readthedocs.io/en/latest/deprecation.html). Note that Python 3.6 is now beyond its end-of-life, i.e. the Python team is no longer publishing version 3.6.X security updates. We do not have plans to re-introduce support for older Python versions into the codebase. If you want to use JAX with older Python versions, the best option would be to install previous releases that are compatible. For Python 3.6, that would be something like this:
```
pip install jax==0.2.17 jaxlib==0.1.69 -f https://storage.googleapis.com/jax-releases/jax_releases.html
```
I hope that helps!

**Reply by JiepingLLL** [reply]
My environment: python3.6
Need to install jax==0.2.17; jaxlib==0.1.69
But I need to build the jaxlib myself and not install it directly with pip. Now there are two problems, one is that I downloaded bazel and added it to bashrc, but when I execute the command "python build/build.py", I get an error "ERROR: Client cwd'/../.. /jax/build' is not inside workpce '/../../jax' b' ' cannot find or download bazel. Please install bazel". This leaves me wondering what to do.
The second is that I see "modify the WORKSPACE file in the root of the JAX source tree to point to a different TensorFlow tree." How to modify the workspace content, I see that the content inside is roughly the same as the tensorflow workspace file the same. I'm so desperate to get your help, it's so important to me.

**Reply by jakevdp** [reply]
Let's move discussion to https://github.com/google/jax/issues/11560
